.github/workflows/ci.yml

```diff
@@ -15,12 +15,17 @@ on:
 jobs:
   build:
     runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: false
+      matrix:
+        JAVA_VERSION: [ '8', '11']
```

Comment:
also remove space from "[ '8'"

Reply:
I have removed the space, but I don't know why git invalid